sudo.ps1

```diff
@@ -22,8 +27,8 @@ function sudo_do($parent_pid, $dir, $cmd) {
 	$kernel::attachconsole($parent_pid)
 		
 	$p = new-object diagnostics.process; $start = $p.startinfo
-	$start.filename = "powershell.exe"
-	$start.arguments = "-noprofile $cmd`nexit `$lastexitcode"
+	$start.filename = "$powershellExe"
+	$start.arguments = "-noprofile -Command $cmd`nexit `$lastexitcode"
```

Comment:
You could use $commandPrefix or similar here as well such that major-versions less equals 5 ain't broken:

    	$start.arguments = "-noprofile $commandPrefix $cmd`nexit `$lastexitcode"

Reply:
Thanks for your suggestions.
But scoop seems to support powershell major version greater than 5 only.

Reply:
Sorry for the confusion on my side, just see ll. 6 in the original sudo.ps1 file to see what I wanted to say.
I definitely mean to support both the `-Command` prefixed and the non-prefixed syntax regardless of major version.